src/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -120,5 +121,57 @@ public void WriteToReadOnlyFile()
                 File.SetAttributes(path, FileAttributes.Normal);
             }
         }
+
+        [Fact]
+        public void EmptyFile_ReturnsEmptyArray()
+        {
+            string path = GetTestFilePath();
+            File.Create(path).Dispose();
+            Assert.Equal(0, File.ReadAllBytes(path).Length);
+        }
+
+        [Theory]
+        [PlatformSpecific(TestPlatforms.Linux)]
```

Comment:
Similar test for OSX maybe?

Reply:
Do you know of any file paths that present this behavior on macOS?